### DISCUSSION TITLE
[Enhancement] [descriptions-item] [Feature Request] descriptions-item新增visible属性 / Add a new attribute [visible] for descriptions-item

### DISCUSSION THREAD

**Discussion by michaelxuzhi** [element-plus/element-plus#11490] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `descriptions-item`

## Description

是否考虑给descriptions-item加一个visible属性，将该descriptions-item隐藏。/ Concidering of add a new attribute [visible] for descriptions-item, hide it when [visible='false'].
```vue
<el-descriptions-item v-for="(val, key) in CarInfo" :key="key" :label="key" :visible="key !== 'BMW'">
</el-descriptions-item>
```

### Additional comments

_(empty)_

**Comment by tolking** [comment]
I think it would be better to filter `CarInfo` by manually.

**Reply by michaelxuzhi** [reply]
How if `CarInfo` has properties like `_id: '123'` and `carName: 'carA'` , i don't want to show `_id`  in  `el-descriptions-item` ,but when click `name` item, show its _id content.
```js
CarInfo:{
    _id : '123',   // don't render in el-descriptions-item
    carName: 'BMW' // when click this el-descriptions-item, show _id content
}
```

**Reply by btea** [reply]
[example](https://element-plus.run/#eyJBcHAudnVlIjoiPHNjcmlwdCBzZXR1cD5cbmltcG9ydCB7IHJlYWN0aXZlIH0gZnJvbSAndnVlJ1xuY29uc3QgY2FySW5mbyA9IHJlYWN0aXZlKFtcbiAgeyBrZXk6ICdfaWQnLCBuYW1lOiAnMTIzJywgc2hvdzogZmFsc2UgfSxcbiAgeyBrZXk6ICdNTlcnLCBuYW1lOiAnY2FyTmFtZScsIHNob3c6IHRydWV9XG5dKVxuY29uc3Qgc3dpdGNoSXRlbSA9IChpdGVtKSA9PiB7XG4gIGNvbnN0IGtleSA9IGl0ZW0ua2V5XG4gIGlmIChrZXkgPT09ICdNTlcnKSB7XG4gICAgY29uc3QgaXRlbSA9IGNhckluZm8uZmluZChpdGVtID0+IGl0ZW0ua2V5ID09PSAnX2lkJylcbiAgICBpdGVtLnNob3cgPSB0cnVlXG4gIH1cbn1cbjwvc2NyaXB0PlxuPHRlbXBsYXRlPlxuICA8ZWwtZGVzY3JpcHRpb25zIHRpdGxlPVwiVXNlciBJbmZvXCI+XG4gICAgPHRlbXBsYXRlIHYtZm9yPVwiaXRlbSBpbiBjYXJJbmZvXCIgOmtleT1cIml0ZW0ua2V5XCIgPlxuICAgIFx0PGVsLWRlc2NyaXB0aW9ucy1pdGVtIDpsYWJlbD1cIml0ZW0ua2V5XCIgdi1pZj1cIml0ZW0uc2hvd1wiPlxuICAgICAgXHQ8c3BhbiBAY2xpY2s9XCJzd2l0Y2hJdGVtKGl0ZW0pXCI+XG4gICAgICAgIFx0e3tpdGVtLm5hbWV9fSAgXG4gICAgICAgIDwvc3Bhbj5cbiAgICAgIDwvZWwtZGVzY3JpcHRpb25zLWl0ZW0+XG4gICAgPC90ZW1wbGF0ZT5cblx0PC9lbC1kZXNjcmlwdGlvbnM+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0X21hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJfbyI6e319)
